internal/pkg/plugin/trello/delete.go

```diff
@@ -0,0 +1,7 @@
+package trello
+
+// Delete remove trello.
+func Delete(options map[string]interface{}) (bool, error) {
+
+	return true, nil
```

Comment:
Why is there no implementation? Is this by design?

Reply:
In the previous discussion, it was said that this board cannot be deleted because someone might be using it